### ISSUE TITLE
Fedora packaging: xrdb obsoletes xorg-x11-server-utils

### ISSUE THREAD

**Issue by mjlbach** [Xpra-org/xpra#3096]
I recently updated to the fedora 34 beta. When I try to install the latest version of xpra, I get  the following dependency conflict (gnome3 depends on xrdb)

```
sudo wget -O /etc/yum.repos.d/xpra.repo https://xpra.org/repos/Fedora/xpra-beta.repo
❯ sudo dnf install xpra
Last metadata expiration check: 0:13:36 ago on Sun 25 Apr 2021 11:17:05 AM PDT.
Error:
 Problem: problem with installed package pipewire-pulseaudio-0.3.26-1.fc34.x86_64
  - package pipewire-pulseaudio-0.3.26-1.fc34.x86_64 conflicts with pulseaudio provided by pulseaudio-14.2-3.fc34.x86_64
  - package pipewire-pulseaudio-0.3.26-1.fc34.x86_64 conflicts with pulseaudio-daemon provided by pulseaudio-14.2-3.fc34.x86_64
  - package pulseaudio-14.2-3.fc34.x86_64 conflicts with pulseaudio-daemon provided by pipewire-pulseaudio-0.3.26-1.fc34.x86_64
  - package pipewire-pulseaudio-0.3.25-1.fc34.x86_64 conflicts with pulseaudio provided by pulseaudio-14.2-3.fc34.x86_64
  - package pipewire-pulseaudio-0.3.25-1.fc34.x86_64 conflicts with pulseaudio-daemon provided by pulseaudio-14.2-3.fc34.x86_64
  - package pulseaudio-14.2-3.fc34.x86_64 conflicts with pulseaudio-daemon provided by pipewire-pulseaudio-0.3.25-1.fc34.x86_64
  - package xpra-4.1.1-1.fc34.x86_64 requires pulseaudio, but none of the providers can be installed
  - conflicting requests
  - package xpra-4.1.2-10.r78.fc34.x86_64 requires python3-xpra-server = 4.1.2-10.r78.fc34, but none of the providers can be installed
  - package xpra-4.1.3-10.r1.fc34.x86_64 requires python3-xpra-server = 4.1.3-10.r1.fc34, but none of the providers can be installed
  - package xpra-4.1.2-10.r14.fc34.x86_64 requires python3-xpra-server = 4.1.2-10.r14.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28636.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28636.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28638.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28638.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28641.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28641.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28667.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28667.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28703.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28703.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28760.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28760.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28834.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28834.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28862.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28862.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28877.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28877.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28878.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28878.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28886.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28886.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28905.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28905.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28916.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28916.fc34, but none of the providers can be installed
  - package xpra-4.2-10.r28924.fc34.x86_64 requires python3-xpra-server = 4.2-10.r28924.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.1.2-10.r78.fc34.x86_64 requires xpra-common-server = 4.1.2-10.r78.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.1.3-10.r1.fc34.x86_64 requires xpra-common-server = 4.1.3-10.r1.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.1.2-10.r14.fc34.x86_64 requires xpra-common-server = 4.1.2-10.r14.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28636.fc34.x86_64 requires xpra-common-server = 4.2-10.r28636.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28638.fc34.x86_64 requires xpra-common-server = 4.2-10.r28638.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28641.fc34.x86_64 requires xpra-common-server = 4.2-10.r28641.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28667.fc34.x86_64 requires xpra-common-server = 4.2-10.r28667.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28703.fc34.x86_64 requires xpra-common-server = 4.2-10.r28703.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28760.fc34.x86_64 requires xpra-common-server = 4.2-10.r28760.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28834.fc34.x86_64 requires xpra-common-server = 4.2-10.r28834.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28862.fc34.x86_64 requires xpra-common-server = 4.2-10.r28862.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28877.fc34.x86_64 requires xpra-common-server = 4.2-10.r28877.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28878.fc34.x86_64 requires xpra-common-server = 4.2-10.r28878.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28886.fc34.x86_64 requires xpra-common-server = 4.2-10.r28886.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28905.fc34.x86_64 requires xpra-common-server = 4.2-10.r28905.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28916.fc34.x86_64 requires xpra-common-server = 4.2-10.r28916.fc34, but none of the providers can be installed
  - package python3-xpra-server-4.2-10.r28924.fc34.x86_64 requires xpra-common-server = 4.2-10.r28924.fc34, but none of the providers can be installed
  - package xpra-common-server-4.1.2-10.r78.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.1.3-10.r1.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.1.2-10.r14.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28636.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28638.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28641.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28667.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28703.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28760.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28834.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28862.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28877.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28878.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28886.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28905.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28916.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - package xpra-common-server-4.2-10.r28924.fc34.noarch requires xorg-x11-server-utils, but none of the providers can be installed
  - problem with installed package xrdb-1.1.1-2.fc34.x86_64
  - installed package xrdb-1.1.1-2.fc34.x86_64 obsoletes xorg-x11-server-utils < 7.7-40 provided by xorg-x11-server-utils-7.7-39.fc34.x86_64
  - package xrdb-1.1.1-2.fc34.x86_64 obsoletes xorg-x11-server-utils < 7.7-40 provided by xorg-x11-server-utils-7.7-39.fc34.x86_64
(try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages)
```

**Comment by totaam** [comment]
Thanks for the heads up!
The fix above has been applied to all supported branches and I've tested it on a Fedora 34 installation with the beta repository (v4.2).

I've also pushed 4.1.3-10.r2 Fedora 34 packages to the stable repository with this fix.

***

I'm not sure why they didn't just have a transition period with `xorg-x11-server-utils` as a metapackage depending on all the new utility packages it got split into. This would have made the transition less painful.
